src/DssSpellCollateralOnboarding.sol

```diff
@@ -31,60 +35,91 @@ contract DssSpellCollateralOnboardingAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmTRiQ3GqjCiRhh1ojzKzgScmSsiwQPLyjhgYSxZASQekj
     //
+    uint256 constant THREE_POINT_FIVE_PCT = 1000000001090862085746321732;
 
     // --- Math ---
+    uint256 constant THOUSAND   = 10 ** 3;
+    uint256 constant MILLION    = 10 ** 6;
+
+    // --- PRE-REQUISITE GENERAL DEPLOYS ---
+    address constant CDP_REGISTRY              = 0xBe0274664Ca7A68d6b5dF826FB3CcB7c620bADF3;
+    address constant PROXY_ACTIONS_CROPPER     = 0xa2f69F8B9B341CFE9BfBb3aaB5fe116C89C95bAF;
+    address constant PROXY_ACTIONS_END_CROPPER = 0xAa61752a5Abf86A527A09546F23FE8bCB8fAB2C4;
+    address constant CROPPER                   = 0x8377CD01a5834a6EaD3b7efb482f678f2092b77e;
 
     // --- DEPLOYED COLLATERAL ADDRESSES ---
+    address constant ETHSTETH                  = 0x06325440D014e39736583c165C2963BA99fAf14E;
+    address constant PIP_ETHSTETH              = 0x2b465de3e69a2EC00158F0E3B4614e3582430ab2;
+    address constant MCD_JOIN_ETHSTETH_A       = 0x036A451114E3835AbEF163A67163B6B376cF2480;
+    address constant MCD_CLIP_ETHSTETH_A       = 0x2Ae099CE87c1A1291953373F660bdEbbdc1928E9;
+    address constant MCD_CLIP_CALC_ETHSTETH_A  = 0x8a4780acABadcae1a297b2eAe5DeEbd7d50DEeB8;
+
+    // --- Other addresses ---
+    address constant STETH_PRICE               = 0x911D7A8F87282C4111f621e2D100Aa751Bab1260;
```

Comment:
I see that the immutables are not public here - https://github.com/makerdao/curve-lp-oracle/blob/master/src/StethPrice.sol#L35
Should consider changing that and redeploying.

Reply:
@kmbarry1

Reply:
I think it is OK

Reply:
Late to the party but with code verification + visible constructor args I really don't think they need to be public.